Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(decodeXwithRC.py:707322): Gdk-CRITICAL **: 09:48:21.150: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
